### ISSUE TITLE
`azure site scale instances` crashes on parsing

### ISSUE THREAD

**Issue by fatso83** [Azure/azure-xplat-cli#2642]
CLI Version: **0.9.18**
OS Type: Mac
Installation via:  npm

Mode: ASM

Description:
`azure site scale instances -v our-company-rest-api-dev` crashes badly

(as a side note you can also see that it does not pick up the hostname, and I have to type it again)

Steps to reproduce:
1) Run `azure site scale  instances -v our-company-rest-api-dev` will output

```
info:    Executing command site scale instances
Web site name: our-company-rest-api-dev
Web site slot [enter for none]:
verbose: Getting site information
verbose: Site enabledHostNames=[our-company-rest-api-dev.azurewebsites.net, our-company-rest-api-dev.scm.azurewebsites.net], hostNames=[our-company-rest-api-dev.azurewebsites.net], hostNameSslStates=[name=our-company-rest-api-dev.azurewebsites.net, sslState=Disabled, name=our-company-rest-api-dev.scm.azurewebsites.net, sslState=Disabled], adminEnabled=true, availabilityState=Normal, sku=Premium, enabled=true, lastModifiedTimeUtc=Wed Mar 16 2016 10:47:05 GMT+0100 (CET), name=our-company-rest-api-dev, repositorySiteName=our-company-rest-api-dev, runtimeAvailabilityState=Normal, uri=https://something-hidden-prod-db3-011.api.azurewebsites.windows.net:454/subscriptions/909478a5-4c67-40a9-9a40-c5819876205f/webspaces/NorthEuropewebspace/sites/our-company-rest-api-dev, serverFarm=Test_alwayson, , , RepositoryUri=https://our-company-rest-api-dev.scm.azurewebsites.net, PublishingUsername=$our-company-rest-api-dev, PublishingPassword=TbeCtZo1cgcCByup1ydR4lPSlrX9qGgBncDi9GuTQpfqjKY5HXyX2FvoT2H1, state=Running, usageState=Normal, webSpace=NorthEuropewebspace
error:   Cannot read property 'charAt' of undefined
info:    Error information has been recorded to /Users/carl-erik.kopseng/.azure/azure.err
verbose: TypeError: Cannot read property 'charAt' of undefined
  <<< async stack >>>
    at ___ (/usr/local/lib/node_modules/azure-cli/lib/commands/asm/site.scale.js:88:45)
    at ___ (/usr/local/lib/node_modules/azure-cli/node_modules/streamline/lib/callbacks/runtime.js:111:13)
    at getSite__8 (/usr/local/lib/node_modules/azure-cli/lib/commands/asm/websites/websitesclient.js:519:20)
  at __2 (/usr/local/lib/node_modules/azure-cli/lib/commands/asm/site.scale.js:84:37)
  <<< raw stack >>>
    at ___ (/usr/local/lib/node_modules/azure-cli/lib/commands/asm/site.scale.js:88:45)
    at ___ (/usr/local/lib/node_modules/azure-cli/node_modules/streamline/lib/callbacks/runtime.js:111:13)
    at __$getSite__8 (/usr/local/lib/node_modules/azure-cli/lib/commands/asm/websites/websitesclient.js:519:20)
    at __tryCatch (/usr/local/lib/node_modules/azure-cli/node_modules/streamline/lib/callbacks/runtime.js:150:4)
    at ___ (/usr/local/lib/node_modules/azure-cli/lib/commands/asm/websites/websitesclient.js:517:179)
    at ___ (/usr/local/lib/node_modules/azure-cli/lib/commands/asm/websites/websitesclient.js:517:112)
    at __tryCatch (/usr/local/lib/node_modules/azure-cli/node_modules/streamline/lib/callbacks/runtime.js:150:4)
    at ___ (/usr/local/lib/node_modules/azure-cli/lib/commands/asm/websites/websitesclient.js:517:67)
    at __$getSite__8 (/usr/local/lib/node_modules/azure-cli/lib/commands/asm/websites/websitesclient.js:517:33)
    at __tryCatch (/usr/local/lib/node_modules/azure-cli/node_modules/streamline/lib/callbacks/runtime.js:150:4)
error:   site scale instances command failed
```

Error stack trace:
```
2016-03-16T11:07:51.019Z:
{ [TypeError: Cannot read property 'charAt' of undefined]
  stack: [Getter/Setter],
  __frame:
   { name: '__2',
     line: 69,
     file: '/usr/local/lib/node_modules/azure-cli/lib/commands/asm/site.scale.js',
     prev: undefined,
     calls: 3,
     active: false,
     offset: 15,
     col: 36 },
  rawStack: [Getter] }
TypeError: Cannot read property 'charAt' of undefined
  <<< async stack >>>
    at ___ (/usr/local/lib/node_modules/azure-cli/lib/commands/asm/site.scale.js:88:45)
    at ___ (/usr/local/lib/node_modules/azure-cli/node_modules/streamline/lib/callbacks/runtime.js:111:13)
    at getSite__8 (/usr/local/lib/node_modules/azure-cli/lib/commands/asm/websites/websitesclient.js:519:20)
  at __2 (/usr/local/lib/node_modules/azure-cli/lib/commands/asm/site.scale.js:84:37)
  <<< raw stack >>>
    at ___ (/usr/local/lib/node_modules/azure-cli/lib/commands/asm/site.scale.js:88:45)
    at ___ (/usr/local/lib/node_modules/azure-cli/node_modules/streamline/lib/callbacks/runtime.js:111:13)
    at __$getSite__8 (/usr/local/lib/node_modules/azure-cli/lib/commands/asm/websites/websitesclient.js:519:20)
    at __tryCatch (/usr/local/lib/node_modules/azure-cli/node_modules/streamline/lib/callbacks/runtime.js:150:4)
    at ___ (/usr/local/lib/node_modules/azure-cli/lib/commands/asm/websites/websitesclient.js:517:179)
    at ___ (/usr/local/lib/node_modules/azure-cli/lib/commands/asm/websites/websitesclient.js:517:112)
    at __tryCatch (/usr/local/lib/node_modules/azure-cli/node_modules/streamline/lib/callbacks/runtime.js:150:4)
    at ___ (/usr/local/lib/node_modules/azure-cli/lib/commands/asm/websites/websitesclient.js:517:67)
    at __$getSite__8 (/usr/local/lib/node_modules/azure-cli/lib/commands/asm/websites/websitesclient.js:517:33)
    at __tryCatch (/usr/local/lib/node_modules/azure-cli/node_modules/streamline/lib/callbacks/runtime.js:150:4)
```

**Comment by talarari** [comment]
any update on this?

**Comment by fatso83** [comment]
This duplicated #1950 , so closing it. Coincidentally a (supposed) fix has been available since April:  qbikez/azure-xplat-cli#1
